docs/Node_Health_Management.md

```diff
@@ -0,0 +1,262 @@
+```
+# systemctl status atomic-openshift-node -l
+● atomic-openshift-node.service - Atomic OpenShift Node
+   Loaded: loaded (/usr/lib/systemd/system/atomic-openshift-node.service; enabled; vendor preset: disabled)
+  Drop-In: /usr/lib/systemd/system/atomic-openshift-node.service.d
+           └─openshift-sdn-ovs.conf
+   Active: active (running) since Thu 2020-10-29 07:37:31 UTC; 1min 16s ago
+     Docs: https://github.com/openshift/origin
+ Main PID: 2412 (openshift)
+   CGroup: /system.slice/atomic-openshift-node.service
+           └─2412 /usr/bin/openshift start node --config=/etc/origin/node/node-config.yaml --loglevel=2
+
+Oct 29 07:55:00 master0.ocp-support-suad.cp.fyre.ibm.com atomic-openshift-node[2412]: E0225 08:37:31.938263    2412 proxier.go:218] Error flushing userspace chain: error flushing chain "KUBE-NODEPORT-CONTAINER": exit status 1: iptables: No chain/target/match by that name.
+Oct 29 07:55:00 master0.ocp-support-suad.cp.fyre.ibm.com atomic-openshift-node[2412]: I0225 08:37:31.938540    2412 node.go:256] Started Kubernetes Proxy on 0.0.0.0
+Oct 29 07:55:00 master0.ocp-support-suad.cp.fyre.ibm.com systemd[1]: Started Atomic OpenShift Node.
+Oct 29 07:55:00 master0.ocp-support-suad.cp.fyre.ibm.com atomic-openshift-node[2412]: I0225 08:37:31.956248    2412 proxier.go:352] Setting endpoints for "default/kubernetes:dns-tcp" to [192.168.200.50:53 192.168.200.51:53 192.168.200.52:53]
+Oct 29 07:55:00 master0.ocp-support-suad.cp.fyre.ibm.com atomic-openshift-node[2412]: I0225 08:37:31.956397    2412 proxier.go:352] Setting endpoints for "default/kubernetes:dns" to [192.168.200.50:53 192.168.200.51:53 192.168.200.52:53]
+Oct 29 07:56:00 master0.ocp-support-suad.cp.fyre.ibm.com atomic-openshift-node[2412]: I0225 08:37:31.956434    2412 proxier.go:352] Setting endpoints for "default/kubernetes:https" to [192.168.200.50:8443 192.168.200.51:8443 192.168.200.52:8443]
+Oct 29 07:56:00 master0.ocp-support-suad.cp.fyre.ibm.com atomic-openshift-node[2412]: I0225 08:37:31.956476    2412 proxier.go:429] Not syncing iptables until Services and Endpoints have been received from master
+Oct 29 07:56:00 master0.ocp-support-suad.cp.fyre.ibm.com atomic-openshift-node[2412]: I0225 08:37:31.965155    2412 proxier.go:294] Adding new service "default/kubernetes:https" at 172.50.0.1:443/TCP
+Oct 29 07:56:00 master0.ocp-support-suad.cp.fyre.ibm.com atomic-openshift-node[2412]: I0225 08:37:31.965358    2412 proxier.go:294] Adding new service "default/kubernetes:dns" at 172.50.0.1:53/UDP
+Oct 29 07:56:00 master0.ocp-support-suad.cp.fyre.ibm.com atomic-openshift-node[2412]: I0225 08:37:31.965450    2412 proxier.go:294] Adding new service "default/kubernetes:dns-tcp" at 172.50.0.1:53/TCP
+```
+This will resolve the problem most of the times and ```oc get nodes``` will give the positive result.
+
+**/var/log/messages** can sometimes shed some additional light if the problem is not resolved by restarting the atomic-openshift-node service.
```

Comment:
```suggestion
`/var/log/messages` can sometimes shed some additional light if the problem is not resolved by restarting the atomic-openshift-node service.
```

Cosmetic change, but we have a bigger problem here, my bad, and this `atomic-openshift-node` service is an OKD implementation, I believe. We don't have that in OCP. Either we try to rewrite this section with OCP equivalent, or we just publish this version and quickly try to replace after.